packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -117,8 +145,8 @@ export class InputGroup extends AbstractPureComponent<InputGroupProps, InputGrou
                 [Classes.DISABLED]: disabled,
                 [Classes.READ_ONLY]: readOnly,
                 [Classes.FILL]: fill,
-                [Classes.LARGE]: large,
-                [Classes.SMALL]: small,
+                [Classes.LARGE]: large || size === "large",
+                [Classes.SMALL]: small || size === "small",
```

Comment:
Should we be using `sizeClass()` here?

Reply:
Rebased to include changes and utilities from previous base branches